client/dev-error-overlay/hot-dev-client.js

```diff
@@ -30,6 +29,16 @@ const {
 let hadRuntimeError = false
 let customHmrEventHandler
 export default function connect (options) {
+  // Open stack traces in an editor.
+  ErrorOverlay.setEditorHandler(function editorHandler ({ fileName, lineNumber, colNumber }) {
+    window.fetch(
```

Comment:
I'm thinking this should use `unfetch`, same as `on-demand-entries-client.js`

```
import fetch from 'unfetch'
```

https://github.com/zeit/next.js/blob/canary/client/on-demand-entries-client.js#L4